### PR TITLE
Feat: Add documentation build

### PR DESCRIPTION
**Pull request by oxedions**
Add documentation builder

### CODE REVIEW

File: auto_builder.sh [hunk 1]
```diff
@@ -60,6 +60,7 @@ echo "     5 - Atftp"
 echo "     6 - Powerman"
 echo "     7 - Conman"
 echo "     8 - iPXE roms"
+echo "     11 - Documentation"
 echo " 9 - Exit."
```
Comment (btravouillon) [inline auto_builder.sh:64]: I don't understand the numbering. Proposal:
9 - documentation
Q - Quit

Comment (oxedions) [reply auto_builder.sh:64]: Yeah, I need to update it each time, boring. Q is a nice idea, I don't know why I was focusing on number...

Thanks a lot ! Let me update that.

File: auto_builder.sh [hunk 2]
```diff
@@ -445,7 +446,26 @@ case $value in
        set +x
     ;;
 
-    9) ######################################################################################
+    11) #####################################################################################
+       set -x
+
+        pip3 install sphinx
+        pip3 install sphinx_rtd_theme
```
Comment (btravouillon) [inline auto_builder.sh:453]: Who do you run 2 pip transactions instead of one?

Comment (oxedions) [reply auto_builder.sh]: Yup, can be merged. done.

**Pull request merged**